paf-mvp-cmp/src/controller.ts

```diff
@@ -3,23 +3,16 @@ import { BindingDisplayRandomId, BindingShowRandomIdDiv, BindingThisSiteOnly } f
 import { Log } from '@core/log';
 import { BindingButton, BindingChecked, BindingCheckedMap, BindingElement } from '@core/ui/binding';
 import { Identifier, IdsAndOptionalPreferences, Preferences, PreferencesData } from '@core/model/generated-model';
-import {
-  deleteIdsAndPreferences,
-  getIdsAndPreferences,
-  getNewId,
-  refreshIdsAndPreferences,
-  removeCookie,
-  ShowPromptOption,
-  signPreferences,
-  updateIdsAndPreferences,
-} from '@frontend/lib/paf-lib';
 import { Marketing, Model } from './model';
 import { PafStatus } from '@frontend/enums/status.enum';
 import { View } from './view';
 import { getCookieValue } from '@frontend/utils/cookie';
 import { Cookies, getPrebidDataCacheExpiration } from '@core/cookies';
 import { TcfCore } from './tcfcore';
 import { ILocale } from './ILocale';
+import { Window } from '@frontend/global';
+
+const OneKeyLib = (<Window>window).PAF;
```

Comment:
This change impacts many lines in the file and is unrelated to the architecture changes we aim to do. We must retain ourselves concerning refactoring so that this PR doesn't become unnecessary big and not easy to review.

Reply:
No, it's not cosmetic: the calls to `paf-lib` go through the global `PAF` object, while before they were based on imports (and the `paf-lib` included in build).
This is a key change as it guarantees that properties (like `status` and `unpersistedIds` are accurate)